templates/.github/workflows/go-check.yml

```diff
@@ -5,6 +5,8 @@ jobs:
   unit:
     runs-on: ubuntu-latest
     name: All
+    env:
+      RUNGOGENERATE: false
```

Comment:
If we drop our env variables in a json file, we can use `env: ${{ fromJson(...) }}` (I think).